dev/io.openliberty.restfulWS.3.0.cdi.3.0_fat/fat/src/io/openliberty/restfulWS30/cdi30/fat/FATSuite.java

```diff
@@ -39,7 +39,7 @@
 })
 public class FATSuite {
     @ClassRule
-    public static RepeatTests r = RepeatTests.withoutModification()
+    public static RepeatTests r = RepeatTests.withoutModificationInFullMode()
```

Comment:
Same issue here.  On Java 8 nothing will run which is not allowed so need the conditional full fat method here as well for without modification.